connectors/connectors.go

```diff
@@ -730,3 +753,118 @@ func StartPeriodic(ctx context.Context, t time.Duration, fn func()) {
 	/* call handler immediately */
 	go handler()
 }
+
+// TODO: move methods below from connectors.go to separated 'utils' file?
+func FormatTimeForStatusMessage(value time.Duration, minRound time.Duration) string {
+	h := value.Hours()
+	m := value.Minutes()
+	s := value.Seconds()
+	if h > 3 || minRound > time.Minute {
+		if h < 73 || minRound == time.Hour {
+			return fmt.Sprintf("%.0f hour(s)", h)
+		} else {
+			d, _ := time.ParseDuration(fmt.Sprintf("%.0fh", h))
+			days := d.Hours() / 24
+			return fmt.Sprintf("%.0f day(s)", days)
+		}
+		// extend with months, years
+	}
+	if m > 3 || minRound > time.Second {
+		return fmt.Sprintf("%.0f minute(s)", m)
+	}
+	return fmt.Sprintf("%.0f second(s)", s)
+}
+
+func addServiceStatusText(patternMessage string, service *transit.DynamicMonitoredService) {
```

Comment:
Please handle -1 special case as discussed to not show W or C value

Reply:
Done in commit https://github.com/gwos/tcg/pull/208/commits/9d1cf264efc0279fb06a50347f1ef2b27e89c580
If both thresholds are provided it adds: [W/C=x/y]only warning threshold is provided it adds [WARN=x]; if only critical threshold is provided it adds [CRITICAL=y]; if no thresholds (both are -1) it doesn't add thresholds text

Reply:
sounds right